libs/assets/src/lib/asset-option.tsx

```diff
@@ -0,0 +1,15 @@
+import { Option } from '@vegaprotocol/ui-toolkit';
+import type { AssetFieldsFragment } from './__generated__/Asset';
+
+export const AssetOption = ({ asset }: { asset: AssetFieldsFragment }) => {
+  return (
+    <Option key={asset.id} value={asset.id}>
+      <div className="flex flex-col items-start">
+        <span>{asset.name}</span>
+        <div className="text-[10px] font-mono w-full text-left break-all">
```

Comment:
**praise (non-blocking)**: I've just put the same `break-all` in the block explorer - previously these were `overflow: scroll` but I much prefer this behaviour too